gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonLongRunningClient.java

```diff
@@ -35,56 +35,108 @@
 import com.google.api.gax.rpc.LongRunningClient;
 import com.google.api.gax.rpc.TranslatingUnaryCallable;
 import com.google.api.gax.rpc.UnaryCallable;
+import com.google.longrunning.CancelOperationRequest;
+import com.google.longrunning.DeleteOperationRequest;
+import com.google.longrunning.GetOperationRequest;
+import com.google.longrunning.Operation;
+import com.google.protobuf.Empty;
 
 /**
  * Implementation of LongRunningClient based on REST transport.
  *
  * <p>Public for technical reasons. For internal use only.
  */
 @InternalApi
-public class HttpJsonLongRunningClient<RequestT, OperationT> implements LongRunningClient {
+public class HttpJsonLongRunningClient implements LongRunningClient {
 
-  private final UnaryCallable<RequestT, OperationT> operationCallable;
-  private final OperationSnapshotFactory<RequestT, OperationT> operationSnapshotFactory;
-  private final PollingRequestFactory<RequestT> pollingRequestFactory;
+  private final UnaryCallable getOperationCallable;
+  private final UnaryCallable cancelOperationCallable;
+  private final UnaryCallable deleteOperationCallable;
 
-  public HttpJsonLongRunningClient(
-      UnaryCallable<RequestT, OperationT> operationCallable,
-      OperationSnapshotFactory<RequestT, OperationT> operationSnapshotFactory,
-      PollingRequestFactory<RequestT> pollingRequestFactory) {
-    this.operationCallable = operationCallable;
-    this.operationSnapshotFactory = operationSnapshotFactory;
-    this.pollingRequestFactory = pollingRequestFactory;
+  private HttpJsonLongRunningClient(
+      UnaryCallable getOperationCallable,
+      UnaryCallable cancelOperationCallable,
```

Comment:
Are `operationSnapshotFactory` and `pollingRequestFactory` used previously?

Reply:
Those were created manually as part of the MethodDescriptors: https://github.com/googleapis/gapic-generator-java/blob/291fce42ae4fa9edc8370334593c455ea8b1b48a/gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/longrunning/stub/HttpJsonOperationsStub.java#L133-L137

The MethodDescriptors that I just generated locally do not have those added in: https://github.com/lqiu96/google-cloud-speech-v1-java/blob/a6885c39376e83037822be35c28b285e32763297/gapic-google-cloud-speech-v1-java/src/main/java/com/google/cloud/speech/v1/stub/HttpJsonSpeechStub.java#L183-L215

Reply:
This was the original generation PR: https://github.com/googleapis/gapic-generator-java/commit/e295459b4fa14cf587b8b4692a29cc8614b51d1c#diff-8629cab86f0bb27abb5f5642ffd8614675d956d70a114c9657cc368ddec7adaf which doesn't have those.